### ISSUE TITLE
Is Orator still under active development?

### ISSUE THREAD

**Issue by thmsklngr** [sdispater/orator#369]
Hi,
currently I'm testing different ORMs just for fun and while searching for potential test candidates aside SA I stumbled over Orator. It looks pretty promising, like PonyORM, but I see that the last commit was done somewhere in January.
Therefore I'd like to ask if it's still under active development or not.
Many thanks in advance, regards, Thomas

**Comment by ghost** [comment]
It's a bit unclear at the moment @mosesontheweb ~thought currently it looks like its dead.~* I've been corrected! See answer by @josephmancuso below!

**Comment by josephmancuso** [comment]
@mosesontheweb @alexpdr 

Hey people,

Orator is not "dead" (sdispater still uses it himself) but the project will not be recevng fixes or features as far as I know and deff not in the near future. I've talked with sdispater about taking over the project but I have decided, with the help of some Masonite maintainers, to build our own ORM called (for now) Masonite ORM from scratch.

This ORM will be pretty much a drop in replacement for Orator. So far I have swapped a few projects over to use it by simply replacing the parent classes and the imports.

I wouldn't say it is ready for production servers yet though. It is still in version 0.4 of beta but the life of the project is not dead. Masonite ORM will become the new Orator.

You can find the new repo here: https://github.com/MasoniteFramework/orm

**Comment by ghost** [comment]
@josephmancuso thank you for clarifying and apologies for providing bad info!

**Comment by josephmancuso** [comment]
@alexpdr its back up

**Comment by Larsene** [comment]
By why ? Orator is a wonderful orm ! why to abandon it ?

**Comment by josephmancuso** [comment]
The original author has a lot of other really amazing pieces of software taking up his time like pendulum and poetry

**Comment by ghost** [comment]
He _**really**_ does make a lot of fantastic python tools and frameworks. It's amazing to see what he has managed to accomplish. 

I view Orator as python's answer to PHP's Eloquent ORM; clean, easy to understand, easy to use. From that POV it makes sense to move Orator to Masonite (which can only be described as Python's Laravel). Especially if it means that it will get more attention and regular updates! :smile:

**Comment by Larsene** [comment]
you're right. in fact i'm also using pendulum and poetry. Not only this python libraries are wonderful, but the website who hosted them are beautiful and the docs are so useful. But, Orator is great too ! :-)

**Comment by MikeLP** [comment]
Is it hard just to review and close some PR's. It won't take so much time.

**Comment by JarriqTheTechie** [comment]
> @mosesontheweb @alexpdr
> 
> Hey people,
> 
> Orator is not "dead" (sdispater still uses it himself) but the project will not be recevng fixes or features as far as I know and deff not in the near future. I've talked with sdispater about taking over the project but I have decided, with the help of some Masonite maintainers, to build our own ORM called (for now) Masonite ORM from scratch.
> 
> This ORM will be pretty much a drop in replacement for Orator. So far I have swapped a few projects over to use it by simply replacing the parent classes and the imports.
> 
> I wouldn't say it is ready for production servers yet though. It is still in version 0.4 of beta but the life of the project is not dead. Masonite ORM will become the new Orator.
> 
> You can find the new repo here: https://github.com/MasoniteFramework/orm

Great!!
